Dockerfile

```diff
@@ -1,11 +1,22 @@
-FROM ruby:alpine
+FROM ruby:alpine AS builder
 
 RUN mkdir -p /share
+RUN apk add --update build-base git-lfs
+
+COPY . /build
+RUN cd /build && \
+    bundle install && \
+    gem build inspec_tools.gemspec -o inspec_tools.gem
+
+
+FROM ruby:alpine
+
 RUN apk add --update build-base
 
-COPY ./ ./
-RUN gem build inspec_tools.gemspec
-RUN gem install inspec_tools --pre --no-document
+COPY --from=builder /build /build
+RUN cd build && \
```

Comment:
Gem install will take a path, so you can just say `gem install build/inspec_tools.gem`